qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,33 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_freeze_frozen(self, test, params, env):
+        """
+        Freeze the frozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        self.gagent.fsfreeze()
+        error_context.context("Freeze the frozen FS", logging.info)
+        try:
+            self.gagent.fsfreeze(check_status=False)
+        except guest_agent.VAgentCmdError as e:
+            expected = ("The command guest-fsfreeze-freeze has been disabled "
+                        "for this instance")
+            if expected in str(e):
+                logging.debug(e)
+                self.gagent.fsthaw()
```

Comment:
It should present at the `finally` clause of this `try` clause, for example:
```python
try:
    ...
finally:
    if self.gagent.get_fsfreeze_status() == self.gagent.FSFREEZE_STATUS_FROZEN:
        self.gagent.fsthaw(check_status=False)
```

Reply:
@luckyh Hi, xuhan, what do you mean here? You mean I should put line915 into finally clause?
But even if put it in finally clause, line915 also will be executed only after "if expected in e.edata". Because, other scenarios are test.fail.